src/configure/helper/graphHelper.ts

```diff
@@ -142,70 +142,62 @@ export class GraphHelper {
             deserializationMapper: null,
             serializationMapper: null
         })
-        .then((data) => {
-            return <AadApplication>{
-                appId: data.appId,
-                secret: secret
-            };
-        });
+            .then((data) => {
```

Comment:
I think you have a setting which causes these whitespacs before then/catch. Do you prefer this setting. To me it seems mis-formatted